pymc3/step_methods/smc.py

```diff
@@ -419,9 +408,9 @@ def resample(self):
         return outindx
 
 
-def ATMIP_sample(n_steps, step=None, start=None, homepath=None, chain=0, stage=0, n_jobs=1,
+def sample_SMC(n_steps, step=None, start=None, homepath=None, chain=0, stage=0, n_jobs=1,
```

Comment:
We're at 2 vs 1 currently (@aseyboldt, @twiecki and @fonnesbeck). @junpenglao @ColCarroll?